tidy3d/plugins/invdes/optimizer.py

```diff
@@ -80,10 +87,19 @@ def display_fn(self, result: InverseDesignResult, step_index: int) -> None:
     def _initialize_result(self, params0: anp.ndarray = None) -> InverseDesignResult:
```

Comment:
should we just change the API to accept `parameter_spec` as a required argument? and then deprecation warning if `params0` passed?

Reply:
sure